lighthouse-core/test/fraggle-rock/scenarios/stringified-replay-test-pptr.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {promises as fs} from 'fs';
+import {promisify} from 'util';
+import {execFile} from 'child_process';
+
+import {stringify} from '@puppeteer/replay';
+
+import {LH_ROOT} from '../../../../root.js';
+import LighthouseStringifyExtension from '../../../fraggle-rock/replay/stringify-extension.js';
+import {getAuditsBreakdown, createTestState} from './pptr-test-utils.js';
+import {readJson} from '../../test-utils.js';
+
+const execFileAsync = promisify(execFile);
+const desktopReplayJson = readJson(`${LH_ROOT}/lighthouse-core/test/fixtures/fraggle-rock/replay/desktop-test-flow.json`);
+const mobileReplayJson = readJson(`${LH_ROOT}/lighthouse-core/test/fixtures/fraggle-rock/replay/mobile-test-flow.json`);
+const FLOW_JSON_REGEX = /window\.__LIGHTHOUSE_FLOW_JSON__ = (.*);<\/script>/;
+
+describe('Running the stringified output script', function() {
+  // eslint-disable-next-line no-invalid-this
+  this.timeout(60_000);
```

Comment:
if you don't have long beforEach, I think you can just do `).timeout(...)` at end of describe call ( I think that doesn't apply to beforeEach? idk)

Reply:
That didn't work for me. I had to do `this.timeout(60_000)`.